lib/tasks/after_tasks.rake

```diff
@@ -433,10 +433,10 @@ namespace :After do
   end
 
 
-  desc "Clean up work URLs for abuse reports from the last month"
-  task(:clean_abuse_report_work_urls => :environment) do
+  desc "Clean up URLs for abuse reports from the last month"
+  task(:clean_abuse_report_urls => :environment) do
```

Comment:
Is this something that should be run periodically ? or is it just something we run once on deploy ?